transducers.js

```diff
@@ -4,7 +4,7 @@
 var symbolExists = typeof Symbol !== 'undefined';
 
 var protocols = {
-  iterator: symbolExists ? Symbol.iterator : '@@iterator'
+  iterator: symbolExists && Symbol.iterator || '@@iterator'
```

Comment:
For sure, I saw that line when going back over it the other day and I honestly don't know *what* I was thinking there :upside_down_face: